### PR TITLE
Add NLL one hot loss#40

### PR DESCRIPTION
**Pull request by myychal**

### CODE REVIEW

File: brats/losses.py [hunk 1]
```diff
@@ -34,3 +35,69 @@ def __call__(self, prediction: torch.Tensor,
             torch.Tensor: Computed Dice Loss
         """
         return torch.sum(1 - F.dice(prediction, target, self.epsilon))
+
+
+class NLLLossOneHot(nn.Module):
```
Comment (szympie) [inline brats/losses.py:40]: With increasing number of loss classes, I think it would be beneficial to create an interface for it.

File: brats/losses.py [hunk 2]
```diff
@@ -34,3 +35,69 @@ def __call__(self, prediction: torch.Tensor,
...
+                Dimensions - (Batch, Channels, Depth, Height, Width)
+        Returns:
+            torch.Tensor: Computed NLL Loss
+        """
+        target = torch.argmax(target, dim=CLASS_DIM)
+        prediction = torch.log(prediction)
+        return self.nll_loss(prediction, target)
+
+
+class DiceWithNLLOneHot(nn.Module):
```
Comment (szympie) [inline brats/losses.py:72]: What do you think about something like this: 
```python
class ComposedLoss(Loss):
    def __init___(self, losses:typing.List[Loss],weights:typing.list[float]):
        usuall_crap
   def __call__(self, prediction: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
        calculate_weighted_sum
```
then we would do:
```python
loss = ComposedLoss([DiceLoss(...), NLLOneHot(...)],[1.0, 1.0])
```

Loss class would be an interface for all loses

File: brats/losses.py (hunk 1 above)
Comment (myychal) [reply brats/losses.py]: Created an interface

File: brats/losses.py (hunk 2 above)
Comment (myychal) [reply brats/losses.py]: Accepted
